packages/core/src/components/ImageViewer/ZoomControls.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useCallback } from 'react';
+import IconAdd from '@airbnb/lunar-icons/src/interface/IconAdd';
+import IconRemove from '@airbnb/lunar-icons/src/interface/IconRemove';
+import Button from '../Button';
+import IconButton from '../IconButton';
+import ButtonGroup from '../ButtonGroup';
+import Dropdown from '../Dropdown';
+import Menu, { Item } from '../Menu';
+import T from '../Translate';
+import useStyles, { StyleSheet } from '../../hooks/useStyles';
+
+const styleSheet: StyleSheet = () => ({
+  controls: {
+    position: 'relative',
+    display: 'inline-block',
+  },
+});
+
+const ZOOM_FACTOR = 0.5;
+export const ZOOM_OPTIONS = [
+  {
+    label: '100%',
+    scale: 1,
+  },
+  {
+    label: '125%',
+    scale: 1.25,
+  },
+  {
+    label: '150%',
+    scale: 1.5,
+  },
+  {
+    label: '200%',
+    scale: 2,
+  },
+];
+
+export type Props = {
+  /** The current scale / zoom level. 1 by default. */
+  scale?: number;
+  /** Callback when scale / zoom changes */
+  onScale: (scale: number) => void;
+};
+
+/** Zoom controls that can be used with an image viewer component */
+export default function ZoomControls(props: Props) {
+  const [styles, cx] = useStyles(styleSheet);
+  const [visible, setVisible] = useState(false);
+  const { onScale, scale = 1 } = props;
+
+  const zoomOptions = ZOOM_OPTIONS.map((zoom: { label: string; scale: number }) => ({
+    ...zoom,
+    handleOnClick: useCallback(() => {
```

Comment:
Hooks can't be used this way: https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

We'll have to remove the `useCallback` here.